examples/integrations/gcp/README.md

```diff
@@ -24,38 +24,42 @@ Run ETL jobs, train and deploy models, and track the performance using this code
 2. Run `./build.sh` to build the container
```

Comment:
nit: Should this be called "build_run.sh", or "run.sh"?

Reply:
I would change it to run.sh

Reply:
yeah, I think I would, since running is the main operation in there. If the containers are already in place it will still run them